material-ui/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Thanks! I'm not too familiar with having code with multiple licenses in a single project, so I have a question. Does the inclusion of the MIT license mean that:

1) the entire project is still Apache 2 licensed, and this MIT license is just the permission notice for using the MIT-licensed rjsf-material-ui theme in the first place, or
2) all code in this folder `material-ui` is MIT licensed, and everything else in react-jsonschema-form is Apache-2 licensed, or
3) all code added in this PR is MIT licensed, and everything else in react-jsonschema-form (including future contributions to the `material-ui` folder) is Apache-2 licensed?

Just want to figure out what exactly adding this license would mean for the project.

Reply:
Mmm, I'm not sure, but I think the option `2`.

To do the thing easier, I'm going to delete de license and leave al project with Apache 2 license 👍